benchmarks/htap/schemas/psql_native/indexes.sql

```diff
@@ -2,7 +2,6 @@ CREATE INDEX idx_customer ON customer (
     c_w_id
```

Comment:
Why all these changes?

Reply:
because indexes are expensive so if you need less columns thats better because then you can keep more of them in memory.

Reply:
So the extra indexes were not needed and made things just go slowly, also for vanilla PSQL right?

Reply:
yes.